test/Test/Pos/BinarySpec.hs

```diff
@@ -29,22 +38,51 @@ spec = describe "Bi" $ do
             binaryTest @(B.FixedSizeInt Int)
             binaryTest @(B.FixedSizeInt Int64)
         describe "TinyVarInt" $ do
-            -- TODO: check that these actually take 1 byte
             describe "1 byte" $ do
+                it "serializing 0 takes 1 byte" $ serializeFail 0 1
                 prop "0" $ binaryEncodeDecode (B.TinyVarInt 0)
+                it "serializing 1 takes 1 byte" $ serializeFail 1 1
                 prop "1" $ binaryEncodeDecode (B.TinyVarInt 1)
+                it "serializing 0x75 takes 1 byte" $ serializeFail 0x75 1
                 prop "0x75" $ binaryEncodeDecode (B.TinyVarInt 0x75)
+                it "serializing 0x79 takes 1 byte" $ serializeFail 0x79 1
                 prop "0x79" $ binaryEncodeDecode (B.TinyVarInt 0x79)
-            -- TODO: check that these actually take 2 bytes
             describe "2 bytes" $ do
+                it "serializing 0x80 takes 2 bytes" $ serializeFail 0x80 2
```

Comment:
I did this, but then it felt that it'd be hard to understand what the property was without taking a look at the function that implements it. Plus, I did it as a top level binding, which means I had to use a type we don't use very often to represent the test, so I just undid that.

With a `let`, that problem disappears.